pkg/apps/application-list.jsx

```diff
@@ -68,7 +68,8 @@ class ApplicationRow extends React.Component {
                     <div>
                         {comp.summary}
                         <div className="alert alert-danger alert-dismissable">
-                            <button className="close"
+                            <button
+className="close"
```

Comment:
This looks really broken. Also, this is a rule which is much more ambiguous than the others  so far -- it's not really harder to read in the old way. Can we revert this one for now and disable the rule, and revisit this in a follow-up?